app/components/elements/forms/nested_component.html.erb

```diff
@@ -1,18 +1,20 @@
-<div class="card my-3">
-  <div class="card-header">
+<fieldset class="my-3" data-controller="nested-form">
```

Comment:
Bootstrap cards aren't a good fit anymore because 1) they don't look like the designs, and 2) a card may not have multiple bodies.